### PR TITLE
changes in Schedule

### PR DESCRIPTION
**Pull request by SilentFlame**
some changes which were discussed in the 25th July meet for 1st and 3rd August-2016

### CODE REVIEW

File: schedule.html [hunk 1]
```diff
@@ -54,14 +54,19 @@
 				<td>Parent's Meet with Faculty</td>
 			</tr>
 			<tr>
-				<td>Parents’ Meeting with Super Seniors</td>
-				<td>3:00 PM - 4:00 PM <br> Himalaya 105</td>
-				<td>An interactive session with IIIT-H Alumni</td>
+				<td>Discussion with Parents of Dual Degree(DD) students </td>
+				<td>2:00 PM - 3:00 PM </td>
```
Comment (gvishal) [inline schedule.html:58]: 105

File: schedule.html [hunk 2]
```diff
@@ -54,14 +54,19 @@
...
+				<td>Parents of Dual Degree students meets Faculty</td>
 			</tr>
 			<tr>
-				<td>Students’ Meeting with Super Seniors</td>
-				<td>4:00 PM - 5:30 PM <br> Himalaya 105</td>
-				<td>An interactive session with IIIT-H Alumni</td>
+				<td>Parents’ Meeting with Super Seniors/ Students Counselling</td>
```
Comment (gvishal) [inline schedule.html]: Separate block and put 105 for counselling session and 205 for alumni.

File: schedule.html [hunk 3]
```diff
@@ -88,13 +93,23 @@
...
 			</tr>
+			<tr>
+				<td>SACB meeting</td>
+				<td>5:30 PM - 6:30 PM</td>
+				<td>First meet of students with Students Academic Counselling Board.</td>
+			</tr>
+			<tr>
+				<td>Meeting with CCC</td>
+				<td>6:45 PM - 7:30 PM</td>
+				<td>meet with the Campus Canine Cell</td>
```
Comment (gvishal) [inline schedule.html]: s/meet/Meet

remove the

File: schedule.html [hunk 4]
```diff
@@ -54,14 +54,19 @@
...
 			<tr>
-				<td>Students’ Meeting with Super Seniors</td>
-				<td>4:00 PM - 5:30 PM <br> Himalaya 105</td>
-				<td>An interactive session with IIIT-H Alumni</td>
+				<td>Parents’ Meeting with Super Seniors/ Students Counselling</td>
+				<td>3:15 PM - 4:15 PM <br> For Parents<br><br> For students <br><br> Venue:TBD </td>
+				<td><br>An interactive session with IIIT-H Alumni and Parents <br> Counselling session for Students.</td>
+			</tr>
+			<tr>
+				<td>Parents Counselling</td>
```
Comment (gvishal) [inline schedule.html]: s/Parents Counselling/meeting with the college counseling body

File: schedule.html (hunk 3 above)
Comment (SilentFlame) [inline schedule.html]: done

File: schedule.html (hunk 2 above)
Comment (SilentFlame) [inline schedule.html]: done

File: schedule.html (hunk 1 above)
Comment (SilentFlame) [inline schedule.html:58]: done

File: schedule.html (hunk 4 above)
Comment (SilentFlame) [inline schedule.html]: done

**Pull request merged**